packages/web/public/static/locales/de/common.json

```diff
@@ -47,6 +47,8 @@
     "bodyOne": "Brauchst du Hilfe? Wir sind für dich da!",
     "bodyTwo": "Bitte sende eine E-Mail an ",
     "bodyThree": " mit einer detaillierten Beschreibung deines Problems und wir werden uns so schnell wie möglich bei dir melden!",
+    "bodyFour": "Have you seen concerning behavior on the platform? Please refer to our Terms of Service and always feel free to report anything you feel may be misaligned with our guidelines:",
```

Comment:
Hi Robin, 

Somehow that little button that let's me copy the English original in a way that you can later implement it easily is missing :(

Reply:
"bodyFour": "Hast du beunruhigendes Verhalten auf der Plattform gesehen? Bitte bezieh dich auf unsere Allgemeinen Geschäftsbedingungen (Terms of Service) und melde alles, von dem du denkst, dass es unseren Richtlinien widerspricht:",